website/docs/r/domain.html.markdown

```diff
@@ -31,8 +31,8 @@ The following arguments are supported:
   will be created.
 * `memory` - (Optional) The amount of memory in MiB. If not specified the domain
   will be created with 512 MiB of memory be used.
-* `running` - (Optional) Use `false` to turn off the instance. If not specified,
-  true is assumed and the instance, if stopped, will be started at next apply.
+* `domain_shutoff` - (Optional) Use `true` to turn off the instance. If not specified,
```

Comment:
i could change this with `domain_shutdown` open for name suggestions

Reply:
This is confusing:
* in the docs, you are replacing `running` with `domain_shutoff`
* however, the option `running` still exists

Why do we need both options?

Reply:
from my POV the problem was already there in the variable `running`.

We are using this property, for doing other operations, like refresh etc. The `running` is kind of internal property that we use in the domain for know if the domain is up or not, and then we do other stuff. 

The `shutoff_domain` var is only an option that we need to say that after creation we need to shutoff.

This is why i created the other var.
 I could create the runnig var in the meta Domain struct. if you want

Reply:
I just looked at the code, and cannot find a reason to keep the `running` option. Setting it in a `.tf` file, won't change anything. Am I missing something or is this option useless?

Reply:
i think the option is useless. i will remove it

Reply:
domain_shutoff or domain_shutdown ?